java-agent/src/nativeMain/kotlin/com/epam/drill/agent/configuration/AgentArguments.kt

```diff
@@ -96,6 +98,16 @@ data class AgentArguments(
             type = packagePrefixes.toType(),
             value = packagePrefixes,
             description = "Configure package prefixes for scanning and instrumentation",
+        ),
+        AgentArguments::sslTruststore.name to AgentParameter(
+            type = sslTruststore.toType(),
+            value = sslTruststore,
+            description = "Configure SSL truststore for admin connection (leave empty to thrust all)",
```

Comment:
1. Change "thrust" to "trust" 😃 
2. Is it the _path_ to truststore, or is it it's name? (if its a _path_, specify it in the description. I.e. "Configure path to SSL truststore")